client/components/Querries.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Collapsible from 'react-collapsible';
+import { Projects, Project, ProjectQuery } from './Interfaces';
+
+export interface IProps {
+    projects: Projects['projects'];
+    test: any;
+}
+
+// eslint-disable-next-line react/function-component-definition
+const Querries: React.FC<IProps> = ({ projects, test }) => (
+    <div>
+        <div>
+            {projects?.map((project: Project) => (
+                <div aria-hidden="true" onClick={() => test(project)}>
+                    <Collapsible
+                        trigger={<span style={{ padding: '10px 50px' }}>{project.name}</span>}
+                        className="projectCard"
+                    >
+                        <div>
+                            {project.queries.map((query: ProjectQuery) => (
+                                <div className="queryProps">
```

Comment:
Is it worth making a query component?

Reply:
WIll need a bit more clarification on this question.